pkgs/desktops/plasma-5/3rdparty/kwin/scripts/parachute.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, mkDerivation, fetchFromGitHub
+, kcoreaddons, kwindowsystem, plasma-framework, systemsettings }:
+
+mkDerivation rec {
+  pname = "parachute";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "tcorreabr";
+    repo = "parachute";
```

Comment:
I believe `"${pname}$` is preferred (though I don't really know why, except that it may be shorter).

Reply:
Hmm, I matched the style of all of the other plugins in the directory

Reply:
``repo = pname`` avoids a string conversion. I have no strong feelings about this and usually don't say anything either way. Just keep it like it is.